src/common/tvgCompressor.cpp

```diff
@@ -472,4 +472,19 @@ size_t b64Decode(const char* encoded, const size_t len, char** decoded)
 }
 
 
+/************************************************************************/
+/* DJB2 Implementation                                                   */
+/************************************************************************/
+
+unsigned long djb2Encode(const char* str)
+{
+    unsigned long hash = 5381;
+    int c;
+
+    while ((c = *str++)) {
+        hash = ((hash << 5) + hash) + c; // hash * 33 + c
+    }
+    return hash;
+}
```

Comment:
We can reduce code line?
```cpp
unsigned long hash = 5381;

while (int c = *str++) {
    hash = ((hash << 5) + hash) + c; // hash * 33 + c
}
return hash;
```

Reply:
Probably, However, rather than just reducing the number of lines of code, we can focus on simplifying the code by reducing its logical complexity.

Reply:
Yes agree, That should be priority!
LGTM so far :)

Reply:
See: https://jellyfish.co/library/code-complexity/

I've seen many cases where thorvg contributors don't pay much attention to the  'degree of nesting', 

It's not good.

Reply:
Thanks, hermet. I'll keep in mind!